languages/nim/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,31 @@
+You're going to write some code to help you cook a brilliant lasagna from your favorite cookbook.
+
+You have five tasks, all related to cooking your recipe.
+
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+You have four tasks, all related to the time spent cooking the lasagna.
+
+1. Define the expected oven time in minutes
```

Comment:
@ErikSchierboom This headings should be actual headings, right?

Reply:
Yep!